components/Sidebar.tsx

```diff
@@ -109,6 +109,7 @@ const getDocsPath = [
   '/overview/use-cases',
   '/overview/code-of-conduct',
   '/overview/faq',
+  'overview/roadmap',
```

Comment:
This path is missing the / characther in the beginning. Please use '/overview/roadmap',

```suggestion
  '/overview/roadmap',
```